### ISSUE TITLE
[Suggestion]: add EMI support for machines recipes (e.g. extractor)

### ISSUE THREAD

**Issue by GordonHollow** [Soaryn/XyCraftTracker#45]
### XyCraft Module

Machines

### Describe the Suggestion

dd EMI support for machines recipes, cause seems like many big mods refuse to continue JEI support now and have all recipes shown in one recipe mod will be good

**Comment by whatthedrunk** [comment]
This would be great for packs like ATM10. We are trying to move fully to EMI if possible.

**Comment by Soaryn** [comment]
I'll look into it, I've been ultimately waiting for something called Gander before looking at other things since it will solve the dilemma of how we present the extractor recipe.

**Comment by Soaryn** [comment]
I at least have the fabricator transfer working in EMI.... 😂

**Comment by Soaryn** [comment]
I had to turn off the EMI integration for now, as it seems it does NOT like sharing plugin space with JEI.

In short, using just JEI it works. As soon as I add any dev integration for EMI, it shuts off all of the JEI categories it seems. So for now, until I get something called `Gander` we are likely holding off on adding EMI integrations. 

Unfortunately, with how the Extractor recipe currently works in JEI, it is not exactly a fun one to recreate in EMI. Hence the wait on `Gander`

**Comment by GordonHollow** [comment]
Well, yes, introducing that change totally disabled this category even if you have JEI in spare.

So you removing that plugin now, and we can see recipes with spare JEI again?

**Comment by Soaryn** [comment]
That is in the `0.7.45` change I just pushed. So EMI integration is on hold 'til later. In comparison, I much appreciate JEI's api much more.

**Comment by GordonHollow** [comment]
Take this to AE2 devs)
no one would use EMI probably, if they don't choose to drop JEI support totally

I mean...if someone make AE2-JEI compatible addon - it's way easier than have all mod devs forced to do EMI support

**Comment by Soaryn** [comment]
They will not make JEI integration as it stands now, but good news! There is a 3rd party addon that provides the AE integrations.

That being said, I dont think "no one would use EMI" is true. Irrelevant of any decision AE has. It does have some interesting features, but none that I personally need.

**Comment by GordonHollow** [comment]
Just found this addon, i checked month ago and it didn't exist yet.
Good news)

**Comment by Sagenseele** [comment]
I know this issue is rather old, but I would appreciate a second approach to the EMI support. It would be very helpful.
